conan.cmake

```diff
@@ -186,14 +186,22 @@ function(conan_cmake_install)
 endfunction()
 
 
+function(conan_cmake_setup_conanfile)
+  parse_arguments(${ARGV})
+  if(ARGUMENTS_CONANFILE)
+    # configure_file will make sure cmake re-runs when conanfile is updated
+    configure_file(${ARGUMENTS_CONANFILE} ${ARGUMENTS_CONANFILE}.junk)
```

Comment:
Hi Sixten. This functionality seems interesting but a couple of issues:
- Tests are broken, could you please check them?
- I don't fully understand what are you trying to do here. ``configure_file`` could be useful, but I am not sure how will you get cmake re-ran, or why do you just remove the .junk file after the command. Most likely some cmake-fu I ignore, could you please explain a little? Thanks!